UI/css/styles.css

```diff
@@ -159,12 +159,14 @@ nav {
   grid-template-columns: repeat(auto-fit, minmax(calc(500px / 2), 1fr));
 }
 .left {
+  margin-top: 6rem;
```

Comment:
Still seems not to be in the middle of the page. Please make sure the margin-top is equal to the margin-bottom or use padding.
![Screenshot from 2020-10-19 09-09-16](https://user-images.githubusercontent.com/39116572/96413497-03e18f00-11ec-11eb-9663-798b667a7d89.png)